Examples/Algorithms/TrackFinding/src/TrackParamsEstimationAlgorithm.cpp

```diff
@@ -56,7 +56,7 @@ Acts::BoundSquareMatrix makeInitialCovariance(
 
       // theta contribution
       variance +=
-          varianceTheta * std::pow(params[Acts::eBoundQOverP] *
+          varianceTheta * std::pow(params[Acts::eBoundQOverP] /
```

Comment:
We probably have different problems, when this happens, but shouldn't we protect us here against a division by zero?

Reply:
I would leave it up to the user to provide valid bound parameters

Reply:
Assertion maybe?

Reply:
Sure - just note that this would apply to many many more places

Reply:
this fits better in https://github.com/acts-project/acts/pull/3683

Reply:
I am fine with having this division unchecked, because the function can not be called accidentally from outside.